addons/web/controllers/main.py

```diff
@@ -1667,8 +1668,8 @@ def get_fonts(self, fontname=None):
                         fonts.append(font)
         else:
             current_dir = os.path.dirname(os.path.abspath(__file__))
-            fonts_directory = os.path.join(current_dir, '..', 'static', 'src', 'fonts', 'sign')
-            font_filenames = sorted(os.listdir(fonts_directory))
+            fonts_directory = Path(current_dir, '..', 'static', 'src', 'fonts', 'sign')
+            font_filenames = sorted([fn.name for fn in fonts_directory.iterdir() if fn.suffix in ('.ttf', '.otf', '.woff', '.woff2')])
```

Comment:
![fancy](https://user-images.githubusercontent.com/7571158/99780235-7ceb4380-2b16-11eb-8fef-1bab6de396c2.gif)

Reply:
Just a test to verify that ci/security does its job :smiley:

Reply:
nice try, but @xmo-odoo had thought of including `pathlib` in the patterns :P